src/view/OutputView.js

```diff
@@ -0,0 +1,25 @@
+class OutputView {
+  static printOneGame(nameList, cars) {
+    for (let i = 0; i < nameList.length; i += 1) {
+      const carOutput = '-'.repeat(cars[i].position);
+      console.log(`${nameList[i]} : ${carOutput}`);
+    }
+    console.log('');
+  }
+
+  static printAllGame(roundResultList, nameList) {
+    roundResultList.forEach(roundResult => {
+      OutputView.printOneGame(nameList, roundResult);
+    });
+  }
```

Comment:
```suggestion
  static printOneGame(name, output) {
    console.log(`${name} : ${output}`);
    console.log('');
  }

  static printAllGame(nameList, outputList) {
    outputList.forEach((output, index) => {
      OutputView.printOneGame(nameList[index], output);
    });
  }
```

참고로 예시에서는 nameList, outputList 두 매개변수를 받고 있지만, 두 배열을 하나로 합친 배열 하나를 매개변수로 받는 것이 좋습니다(Optional).